MIGRATING.md

```diff
@@ -1954,7 +1954,7 @@ All helper functions have been moved into a new `cosmwasm-schema` package.
 - Remove `serde_json` `[dev-dependency]` if there, as cosmwasm-schema will
   handle JSON output internally.
 - Update `examples/schema.rs` to look
-  [more like queue](https://github.com/CosmWasm/cosmwasm/blob/main/contracts/queue/examples/schema.rs),
+  [more like queue](https://github.com/CosmWasm/cosmwasm/blob/main/contracts/queue/src/bin/schema.rs),
```

Comment:
As this is part of the 0.7.2 -> 0.8 migration (very very old), it is probably better to pin the version in the link:
https://github.com/CosmWasm/cosmwasm/blob/v0.8.0/contracts/queue/examples/schema.rs